### PR TITLE
mailchimp subscribe button

### PR DESCRIPTION
**Pull request by sarinnasung**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ class App extends React.Component<{}, State, {}> {
             <Switch>
               <Route exact path="/" render={() => <Redirect to="/home" />} />
               <Route path="/home" render={() => <Home />} />
-              <Route path="/contact-us" render={() => <ContactUs />} />
+              {/* <Route path="/contact-us" render={() => <ContactUs />} /> */}
```
Comment (crchong1) [inline src/App.tsx]: I think we can still keep the contact us page as it contains other info that is useful

File: src/components/Card.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const formatDescription = (description) => {
 };
 
 const Card = ({
-  imgUrl, title, url, description, date
+  imgUrl, title, url, description, date,
```
Comment (crchong1) [inline src/components/Card.tsx:13]: unnecessary change here

File: src/components/Header.tsx [hunk 3]
```diff
@@ -86,11 +86,6 @@ class Header extends Component<{}, {}, {}> {
                   Stories
                 </Link>
               </li>
-              <li className="nav-item my-1 mr-3 ml-2">
```
Comment (crchong1) [inline src/components/Header.tsx]: you don't have to delete

File: src/components/mailchimpbutton.tsx [hunk 4]
```diff
@@ -0,0 +1,190 @@
+import React, { Component, ReactElement } from 'react';
```
Comment (crchong1) [inline src/components/mailchimpbutton.tsx:1]: Can you camel-case this file name to MailchimpButton?

File: src/components/mailchimpbutton.tsx [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+                  placeholder="Your email"
+                  value={email}
+                  onChange={this.handleChangeEmail}
+                  onBlur={this.validateEmail}
+                  required
+                />
+              </div>
+              <div className="buttonmessage">
+                {this.emailMessage()}
+                {' '}
```
Comment (crchong1) [inline src/components/mailchimpbutton.tsx]: why do we have a space here?

File: src/components/mailchimpbutton.tsx [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+      .then((response) => response.json())
+      .then((responseJSON) => {
+        const { status } = responseJSON;
+        if (status === 'SUCCESS') {
+          alert.show(
+            'Thank you for your interest! We will get back to you promptly.',
+          );
+          this.clearInput();
+        } else {
+          alert.show('Failed to submit. Please fill out all fields correctly.');
```
Comment (crchong1) [inline src/components/mailchimpbutton.tsx]: Maybe a more descriptive error message like "Please enter a valid email address"

File: src/static/styles/App.scss [hunk 7]
```diff
@@ -510,6 +511,18 @@ hr {
...
+.signupbutton-size{
+	border-bottom-left-radius: 0rem;
+	border-top-left-radius: 0rem;
+	
+}
+
+.emailbutton1-size{
+	border-bottom-right-radius: 0rem;
+	border-top-right-radius: 0rem;
+
```
Comment (crchong1) [inline src/static/styles/App.scss]: you can remove the space here

File: src/static/styles/App.scss [hunk 8]
```diff
@@ -510,6 +511,18 @@ hr {
 	border-color: white;
 }
 
+.signupbutton-size{
+	border-bottom-left-radius: 0rem;
+	border-top-left-radius: 0rem;
+
```
Comment (crchong1) [inline src/static/styles/App.scss]: remove this space here

File: src/static/styles/App.scss [hunk 9]
```diff
@@ -510,6 +511,18 @@ hr {
 	border-color: white;
 }
 
+.signupbutton-size{
+	border-bottom-left-radius: 0rem;
+	border-top-left-radius: 0rem;
+	
+}
+
+.emailbutton1-size{
```
Comment (crchong1) [inline src/static/styles/App.scss]: can you make the classname a little more descriptive like "mailchimp-email-input-button"

File: src/static/styles/App.scss [hunk 10]
```diff
@@ -556,6 +569,10 @@ hr {
 	color: $footer-text-color-accent;
 }
 
+.signupheading{
```
Comment (crchong1) [inline src/static/styles/App.scss]: signup-heading

File: src/static/styles/App.scss [hunk 11]
```diff
@@ -627,8 +644,10 @@ hr {
 }
 
 .form-purple {
-	border: 0.0625rem solid $primary-theme;
-	border-radius: 0.35rem;
+	border: 0.2rem solid $primary-theme;
```
Comment (crchong1) [inline src/static/styles/App.scss]: do we need to change these border radius widths? is this in the figma?

File: src/components/mailchimpbutton.tsx [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+    if (inputString === 'true') {
+      return 'is-valid';
+    }
+    if (inputString === 'false') {
+      return 'is-invalid';
+    }
+    return '';
+  };
+
+  handleSubmitWithRecaptcha = async (event: any) => {
```
Comment (crchong1) [inline src/components/mailchimpbutton.tsx:76]: delete this whole function as it isn't being used

Comment (sarinnasung) [reply src/components/mailchimpbutton.tsx:76]: The function is being used for the sign-up button on line 161. The purpose of the function is to click on the signup button and display a confirmation message or an error message. If we remove this function we would not be able to get an alert telling us if the email successfully went through.
